apps/docs/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "bandada-docs",
+    "version": "0.0.0",
```

Comment:
At least `0.0.1`?

Reply:
Thanks. I will remove that `version` property because it's not necessary.